vmdb/app/models/ems_refresh/parsers/ps_scripts/get_datastore_inventory.ps1

```diff
@@ -0,0 +1,69 @@
+import-module virtualmachinemanager

```

Comment:
Is this file required?  I do not see it referenced anywhere.  If it does need to be included there are lots of comments that should be removed and the references to C:\tmp need to use proper PS methods to get a temp file.